backend/uclapi/oauth/tests.py

```diff
@@ -12,14 +14,18 @@
 from .app_helpers import generate_random_verification_code
 from .models import OAuthScope, OAuthToken
 from .scoping import Scopes
-from .views import authorise
+from .views import authorise, shibcallback
 
 @uclapi_protected_endpoint(personal_data=True, required_scopes=["timetable"])
 def test_timetable_request(request, *args, **kwargs):
     return JsonResponse({
         "ok": True
     }, custom_header_data=kwargs)
 
+
+def unsign(data, max_age):
```

Comment:
What does this do, and is it needed? It seems like it just confuses `unsign` with `signer.unsign` and raises an Exception?

Reply:
it does exactly that, mocks signer.unsign so it calls SignatureExpired, easiest way to pretend its been 5 minutes right? there might be a more elegant way of doing this with a decorator?

Reply:
Ah! I see what you've done. Okay it's definitely worth commenting on this to explain how it works, but that's very smart.

Reply:
Alright, i'll stick some comments on it explaining the format then